apps/web/app/api/enable-draft/route.ts

```diff
@@ -0,0 +1,201 @@
+import { cookies, draftMode } from "next/headers";
+import { redirect } from "next/navigation";
+import type { NextRequest } from "next/server";
+import { env } from "~/env";
+
+interface VercelJwt {
```

Comment:
i would remove all of these vercel things since we are never going to be on vercel